packages/cli/src/commands/mock.ts

```diff
@@ -20,6 +21,18 @@ const mockCommand: CommandModule = {
           boolean: true,
           default: false,
         },
+        'callback-delay': {
+          description: 'Number of seconds to wait before executing callbacks',
+          default: '0',
+          string: true,
```

Comment:
Now that I think about it, yargs supports array natively and maybe we could simply get rid of the custom parser?